linux/lvm.go

```diff
@@ -168,7 +171,8 @@ func (ls *linuxLVM) CreatePV(name string) (disko.PV, error) {
 		return nilPV, err
 	}
 
-	err = runCommandSettled("lvm", "pvcreate", "--zero=y", path)
+	err = runCommandSettled("lvm", "pvcreate", "--zero=y",
+		fmt.Sprintf("--metadatasize=%dB", pvMetaDataSize), path)
```

Comment:
You specify the pvMetadataSize now when creating the PV; what was it before?

Reply:
> You specify the pvMetadataSize now when creating the PV; what was it before?

good catch.  it defaults to 4MiB.  We have always set it to 128MiB.
There are 2 ways of initializing a pv:

 1. specifically: pvcreate /dev/path
 2. through vgcreate: vgcreate /dev/path

For the second path, we were correctly passing '--metadatasize', but we were *not* doing so here.

So yeah... this is really a separate change that I just snuck. i've separated into its own commit.